ote_sdk/ote_sdk/test_suite/training_tests_actions.py

```diff
@@ -562,15 +565,25 @@ def __call__(self, data_collector: DataCollector, results_prev_stages: OrderedDi
 
 
 # TODO: think about move to special file
-def check_nncf_model_graph(model, path_to_dot):
+def check_nncf_model_graph(
+    model: "NNCFNetwork", path_to_dot: str, force_regen_dot: bool = False
```

Comment:
Why is `force_regen_dot` set to `False`? Personally, I am in the favour of avoiding committing `dot` files to the repo. The tests might fail if the model architecture changes or if a new model is added with missing dot files.